guides/Front-end-strategy.md

```diff
@@ -0,0 +1,44 @@
+# Front-end strategy
+
+### What we use
+####Foundation 6 framework
+Use only required components and libraries. We don't need to use normalize with Foundation 6, it included in foundation-global-styles mixin.
+
+####Grunt
+Use grant tasks for creating svg sprites
+
+####Autoprefixer
+Use Autoprefixer gem or Autoprefixer of [grunt-postcss](https://github.com/nDmitry/grunt-postcss) task in Shopify projects.
+
+####Bower
+Use bower with bower installer for manage libraries and control their versions.
+
+####Front-man
+It includes utilities classes, z-index variables and helpers for sprite and image tags.
```

Comment:
Front-man is a gem that includes some helpers and style files. utilities - it's a file that includes some utility classes '.u-' https://github.com/DVELP/front_man/blob/master/app/assets/stylesheets/front_man/_utilities.scss you can use them in a project.